source/installation/install-software.rst

```diff
@@ -16,76 +16,80 @@ software requirements:
+.. tabs::
 
-        sudo yum install centos-release-scl epel-release
+   .. tab:: CentOS 7
 
-   RHEL 7
-     .. code-block:: sh
+      .. code-block:: sh
 
-        sudo yum install epel-release
-        sudo subscription-manager repos --enable=rhel-server-rhscl-7-rpms
-        # Repository 'rhel-server-rhscl-7-rpms' is enabled for this system.
+         sudo yum install centos-release-scl epel-release
 
-   .. warning::
 
-      For **RedHat** you may also need to enable the *Optional* channel and
-      attach a subscription providing access to RHSCL to be able to use this
-      repository.
+   .. tab:: RockyLinux 8
 
-#. Enable dnf modules and repositories for dependencies **on RHEL/Rocky 8 only**:
+      .. code-block:: sh
 
-    .. code-block:: sh
+         sudo dnf config-manager --set-enabled powertools
+         sudo dnf install epel-release
+         sudo dnf module enable ruby:2.7
+         sudo dnf module enable nodejs:12
 
-       dnf install epel-release
-       dnf module enable ruby:2.7
-       dnf module enable nodejs:14
+   .. tab:: RHEL 7
 
-   **Rocky 8 only**
+      .. warning::
 
-    .. code-block:: sh
+         You may also need to enable the *Optional* channel and
+         attach a subscription providing access to RHSCL to be able to use this
+         repository.
 
-       sudo dnf config-manager --set-enabled powertools
+      .. code-block:: sh
 
-   **RedHat 8 only**
+         sudo yum install epel-release
+         sudo subscription-manager repos --enable=rhel-server-rhscl-7-rpms
 
-    .. code-block:: sh
 
-       sudo subscription-manager repos --enable codeready-builder-for-rhel-8-x86_64-rpms
+   .. tab:: RHEL 8
 
-#. Add Open OnDemand's repository hosted by the `Ohio Supercomputer Center`_:
+      .. warning::
 
-   **RedHat/CentOS/Rocky only**
+         You may also need to enable the *Optional* channel and
+         attach a subscription providing access to RHSCL to be able to use this
+         repository.
 
-    .. code-block:: sh
+      .. code-block:: sh
 
-       sudo yum install https://yum.osc.edu/ondemand/{{ ondemand_version }}/ondemand-release-web-{{ ondemand_version }}-1.noarch.rpm
+         sudo dnf config-manager --set-enabled powertools
+         sudo dnf install epel-releaseq
+         sudo dnf module enable ruby:2.7
+         sudo dnf module enable nodejs:12
+         sudo subscription-manager repos --enable codeready-builder-for-rhel-8-x86_64-rpms
 
-   **Ubuntu only**
 
-    .. code-block:: sh
+2. Add Open OnDemand's repository and install:
 
-       sudo apt install apt-transport-https ca-certificates
-       wget -O /tmp/ondemand-release-web_{{ ondemand_version }}.0_all.deb https://apt.osc.edu/ondemand/{{ ondemand_version }}/ondemand-release-web_{{ ondemand_version }}.0_all.deb
-       sudo apt install /tmp/ondemand-release-web_{{ ondemand_version }}.0_all.deb
-       sudo apt update
+   .. tabs::
 
-#. Install OnDemand and all of its dependencies:
+      .. tab:: RHEL clones
```

Comment:
I don't know if this is the right phrase for RHEL type systems.

Reply:
I was thinking the same thing about sudo, but I feel like we have to be super explicit or we'll get topics on discourse like 'yum/dnf command not found' lol.

Reply:
Oh I meant the tab could say `yum/dnf` instead of `RHEL clones` but the actual commands would all still use `yum` because that exists still in RHEL8 but might be gone in RHEL9, not sure yet.